src/Wizard.js

```diff
@@ -18,6 +18,7 @@ export default class Wizard extends Webform {
    *    - breadcrumbSettings.clickable: true (default) determines if the breadcrumb bar is clickable or not
    *    - buttonSettings.show*(Previous, Next, Cancel): true (default) determines if the button is shown or not
    *    - allowPrevious: false (default) determines if the breadcrumb bar is clickable or not for visited tabs
+   *    - overrideBreadcrumbClickableOption: false (default) determines if breadcrumbClickable overrides breadcrumbSettings.clickable
```

Comment:
Can we call this ```overrideBreadcrumbClickable```?  

Also, you need to add a default in the ```schema```. Otherwise every new panel component will include this configuration.